Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.8-alpine
```

Comment:
Please use `golang:alpine` instead of tying to a specific version. A utility of this scale will only be modified slightly over time. We'd better just stick to the latest version of Go, so that we don't need to keep track of this manually.